zmdocument.lua

```diff
@@ -0,0 +1,20 @@
+function licence (value)
```

Comment:
The indentation is pretty insconsistent in this file. Please stick with either two or four spaces.